src/thor/bidirectional_astar.cc

```diff
@@ -321,6 +327,10 @@ BidirectionalAStar::GetBestPath(valhalla::Location& origin,
   travel_type_ = costing_->travel_type();
   access_mode_ = costing_->access_mode();
 
+  // Set up the expansion geojson
+  if (auto expansion = expansion_.lock())
+    rapidjson::Pointer("/properties/algorithm").Set(*expansion, "bidirectional_astar");
```

Comment:
the `FeatureCollection` itself has properties which say which algorithm was used in the computation of the route